python/tvm/topi/arm_cpu/dense_alter_op.py

```diff
@@ -82,6 +83,31 @@ def _alter_dense(attrs, inputs, tinfos, out_type):
             False,
             transpose_b,
         )
+    elif topi_impl == "dense_gemm.arm_cpu":
+        # Pre-compute transposed weights and convert to a matmul
+        assert isinstance(
+            inputs[1], relay.Constant
+        ), "dense_gemm.arm_cpu requires weights be a Relay Constant"
+
+        weight_dtype = tinfos[1].dtype
+        weight_data = inputs[1].data.numpy()
+        interleaved = weight_data.transpose()
+        encoded_weight = relay.const(interleaved, weight_dtype)
```

Comment:
I think that's a valid recommendation, probably it will also save time, probably we do the same for the sme case, @lhutton1 ?

Reply:
Thanks for raising this, yes I think it would be great if we could also do the same for SME! There is a subtle difference with SME in that adding transpose here will likely be slower than doing a transpose interleave within the schedule if the transpose (inserted here) cannot be fused into the preceding op. Although, since neither functionality exists today, I'm happy with adding the transpose here :)

Reply:
Thanks, @Anndrey24 @lhutton1, I have updated the file as suggested.